### PR TITLE
created and added content to rle details

### PR DESCRIPTION
**Pull request by JasSmith-ch**
- created and added content to rle details
- not sure why but i merged my pull request from last time and its still saying on this request that the changes will be new such as the rlePscList page and the bug fix

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Review by AlisonGriffiths — CHANGES_REQUESTED** [review]
Have a read through - https://design-system.service.gov.uk/components/date-input/

See the various macro options that are available to you that will help you complete this page.

File: locales/cy/rle-details.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+    "rle_details_title": "To be translated",
+    "rle_details_field_title": "To be translated",
+    "rle_details_first_name": "To be translated",
+    "rle_details_middle_names": "To be translated",
+    "rle_details_last_name": "To be translated",
+    "rle_details_dob_label": "To be translated",
+    "rle_details_hint1": "To be translated",
```
[the remark this answers is not in the document]
Comment (PhilipGlover) [reply locales/cy/rle-details.json:8]: present now and english done too

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

File: src/views/router_views/rleDetails/rleDetails.njk [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+      text: i18n.rle_details_last_name
+    },
+    classes: "govuk-!-width-two-thirds",
+    id: "ro-last-name"
+  }) }}
+
+  <label class="govuk-label">{{i18n.rle_details_dob_label}}</label>
+  <div class="govuk-hint">{{i18n.rle_details_hint1}}</div>
+
+  {{ govukDateInput({
```
[the remark this answers is not in the document]
Comment (PhilipGlover) [reply src/views/router_views/rleDetails/rleDetails.njk:51]: items added

File: src/views/router_views/rleDetails/rleDetails.njk [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+  fieldset: {
+    legend: {
+      isPageHeading: true,
+      classes: "govuk-fieldset__legend--l"
+    }
+  },
+  hint: {
+    text: i18n.rle_details_hint2
```
[the remark this answers is not in the document]
Comment (PhilipGlover) [reply src/views/router_views/rleDetails/rleDetails.njk]: added, though is it best to place this at the top of the template?  Wouldn't it be better above the start of the `govukDateInput` block? This is what I see in the start template.

File: src/views/router_views/rleDetails/rleDetails.njk [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+  {{ govukInput({
+    label: {
+      text: i18n.rle_details_last_name
+    },
+    classes: "govuk-!-width-two-thirds",
+    id: "ro-last-name"
+  }) }}
+
+  <label class="govuk-label">{{i18n.rle_details_dob_label}}</label>
+  <div class="govuk-hint">{{i18n.rle_details_hint1}}</div>
```
[the remark this answers is not in the document]
Comment (PhilipGlover) [reply src/views/router_views/rleDetails/rleDetails.njk]: removed

File: src/views/router_views/rleDetails/rleDetails.njk [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+
+  {{ govukDateInput({
+  fieldset: {
+    legend: {
+      isPageHeading: true,
```
[the remark this answers is not in the document]
Comment (PhilipGlover) [reply src/views/router_views/rleDetails/rleDetails.njk]: changed to false

File: src/views/router_views/rleDetails/rleDetails.njk [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+      classes: "govuk-fieldset__legend--l"
```
[the remark this answers is not in the document]
Comment (PhilipGlover) [reply src/views/router_views/rleDetails/rleDetails.njk]: classes removed

File: src/views/router_views/rleDetails/rleDetails.njk [hunk 7]
```diff
@@ -0,0 +1,84 @@
...
+
+  {{ govukInput({
+    label: {
+      text: i18n.rle_details_last_name
+    },
+    classes: "govuk-!-width-two-thirds",
+    id: "ro-last-name"
+  }) }}
+
+  <label class="govuk-label">{{i18n.rle_details_dob_label}}</label>
```
[the remark this answers is not in the document]
Comment (PhilipGlover) [reply src/views/router_views/rleDetails/rleDetails.njk]: removed label

File: src/views/router_views/rleDetails/rleDetails.njk (hunk 3 above)
[the remark this answers is not in the document]
Comment (PhilipGlover) [reply src/views/router_views/rleDetails/rleDetails.njk]: moved ☑️

**Pull request merged**